backend/app/environments/model.py

```diff
@@ -1,35 +1,12 @@
-import uuid
-
-from sqlalchemy import Boolean, Column, Enum, ForeignKey, String
-from sqlalchemy.dialects.postgresql import UUID
-from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import Boolean, Column, String
 
 from app.database.database import Base
 from app.shared.model import BaseORM
 
-from .enums import PlatformTypes
-
 
 class Environment(Base, BaseORM):
     __tablename__ = "environments"
 
```

Comment:
I would add id to environment table to use instead of name as foreign key

Reply:
I planned to add ID but it affects `permissionsets` functionality. I propose to add it in a separate dedicated PR